src/consensus/aft/raft.h

```diff
@@ -2036,7 +2090,7 @@ namespace aft
       auto new_commit_cft_idx = std::numeric_limits<Index>::max();
 
       // Obtain CFT watermarks
-      for (auto& c : configurations)
+      for (auto const& c : configurations)
```

Comment:
I was going to suggest `auto const&` for consistency with the other code, but it turns out we do it this way round in loads of places!

`auto const&` -> 39 results in 16 files
`const auto&` -> 448 results in 112 files

So ignore for now, but we should standardisze that in the enforced coding standard if we can.